stable/artifactory-ha/README.md

```diff
@@ -1436,6 +1436,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.tomcat.connector.maxThreads`         | The max number of connections to Artifactory connector   | `200` |
 | `artifactory.tomcat.connector.extraConfig`         | The max queue length for incoming connections to Artifactory connector  | `'acceptCount="100"'` |
 | `artifactory.systemYaml`                      | Artifactory system configuration (`system.yaml`) as described here - https://www.jfrog.com/confluence/display/JFROG/Artifactory+System+YAML  | `see values.yaml`    |
+| `artifactory.affinity` | Artifactory node affinity | `{}` |
```

Comment:
@rpf3 it should be `artifactory.primary. affinity` and `artifactory.node. affinity` 
please refer [primary](https://github.com/jfrog/charts/blob/master/stable/artifactory-ha/values.yaml#L1126) and [node](https://github.com/jfrog/charts/blob/master/stable/artifactory-ha/values.yaml#L1184)